Hardware/index.md

```diff
@@ -86,3 +90,21 @@ We support PC99-style Intel Architecture Platforms.
 | [PC99 (64-bit)](IA32) | x64  | VT-X           | VT-D  | [FC (without VT-X, VT-D and fastpath)][X64] | Data61         | seL4 Foundation        |
 
 [X64]: /projects/sel4/verified-configurations.html#x64
+
+
+---
+
+## *Unmaintained platforms*
+
+
+### *ARM*
+
+
+| *Platform* | *System-on-chip* | *Core* | *Arch* | *Virtualisation* | *SMMU* | *Verification Status* | *Contributed by* |
+| - | - | - | - | - | - | - | - | - |
+{%- assign sorted = site.pages | sort: 'platform'%}
+{% for page in sorted %}
+{%- if page.arm_hardware and page.Maintained == "No"" -%}
+| *[{{ page.platform }}]({{page.url}}) (**unmaintained**)* | *{{ page.soc}}* | *{{ page.cpu }}* | *{{ page.arch }}* | *{{ page.virtualization }}* | *{{ page.iommu}}* | *{{ page.Status }}* | *{{ page.Contrib }}* |
```

Comment:
I think keeping the `**unmaintained**` as strong and italic is good, but I would leave the rest as default or even make it slightly greyed out or something like that.

Reply:
Have done grey as said above